tests/ignores.txt

```diff
@@ -28,6 +29,34 @@ cranelift+aarch64+macos    traps::start_trap_pretty
 # https://github.com/wasmerio/wasmer/issues/2808
 cranelift+aarch64 spec::skip_stack_guard_page
 llvm+aarch64      spec::skip_stack_guard_page
+cranelift+riscv64 spec::skip_stack_guard_page
+llvm+riscv64      spec::skip_stack_guard_page
+
+# riscv support is still early, function call ABI needs some work
+llvm+riscv64      static_function::llvm::universal
+llvm+riscv64      static_function_with_env::llvm::universal
+llvm+riscv64      static_function_with_results::llvm::universal
```

Comment:
We should document the RISC-V limitations and behaviour somewhere, maybe in a separate file in `docs/`, and also point this out in the release notes.

Reply:
Ok, I added a file, but not sure what should be put inside, if the details are enough or not.

Reply:
I think that's a good intro.

If users have specific questions they can ask and we can amend the docs.

:+1: